packages/notebook-extension/src/trusted.tsx

```diff
@@ -17,16 +17,19 @@ const isTrusted = (notebook: Notebook): boolean => {
     return false;
   }
   const cells = Array.from(model.cells);
+  let total = 0;
+  let trusted = 0;
 
-  const trusted = cells.reduce((accum, current) => {
-    if (current.trusted) {
-      return accum + 1;
-    } else {
-      return accum;
+  for (const current_cell of cells) {
```

Comment:
It would be good to follow the `camelCase` convention here, this is `currentCell`